### ISSUE TITLE
replicating AIBS.ipynb: key errors in /neuronunit/models/reduced.py

### ISSUE THREAD

**Issue by russelljjarvis** [scidash/neuronunit#33]
Hi @rgerkin,
 
I converted the ipynb file to a regular python file `AIBS.py` and execute that file. The error calls when I try to invoke sciunit judge https://github.com/russelljjarvis/sciunitopt/blob/master/AIBS.py#L146
```
 File "AIBS.py", line 180, in <module>
    suite.judge(model)
```
As you can see the `AIBS.py f`ails at `suite.judge(model)`. At first I thought that problem was in `LEMS_2007One.xml`
because when I run `model.results.keys()`. After I run AIBS `dict_keys(['t', 'RS_pop[0]/v'])`
and because the output column:  RS_pop[0]/v is specified in the `LEMS_2007One.xml` file.

However I have now come around to thinking that the problem is in the file:
`/opt/conda/lib/python3.5/site-packages/neuronunit/models/reduced.py", line 28`
As the error output below would suggest.

```
 File "AIBS.py", line 180, in <module>
    suite.judge(model)
  File "/opt/conda/lib/python3.5/site-packages/sciunit/__init__.py", line 430, in judge
    deep_error=deep_error)
  File "/opt/conda/lib/python3.5/site-packages/sciunit/__init__.py", line 302, in judge
    raise score.score # An exception.  
  File "/opt/conda/lib/python3.5/site-packages/sciunit/__init__.py", line 296, in judge
    score = self._judge(model, skip_incapable=skip_incapable)
  File "/opt/conda/lib/python3.5/site-packages/sciunit/__init__.py", line 239, in _judge
    prediction = self.generate_prediction(model)
  File "/opt/conda/lib/python3.5/site-packages/neuronunit/tests/__init__.py", line 468, in generate_prediction
    lookup = self.threshold_FI(model, units)
  File "/opt/conda/lib/python3.5/site-packages/neuronunit/tests/__init__.py", line 507, in threshold_FI
    f(0.0*units)
  File "/opt/conda/lib/python3.5/site-packages/neuronunit/tests/__init__.py", line 500, in f
    n_spikes = model.get_spike_count()
  File "/opt/conda/lib/python3.5/site-packages/neuronunit/capabilities/__init__.py", line 56, in get_spike_count
    spike_train = self.get_spike_train()
  File "/opt/conda/lib/python3.5/site-packages/neuronunit/models/reduced.py", line 40, in get_spike_train
    vm = self.get_membrane_potential(rerun=rerun, **run_params)
  File "/opt/conda/lib/python3.5/site-packages/neuronunit/models/reduced.py", line 28, in get_membrane_potential
    v = np.array(self.results['v'])
KeyError: 'v'
```

A temporary fix for the file /opt/conda/lib/python3.5/site-packages/neuronunit/models/reduced.py#L28
which seems to work is:
```
for rkey in self.results.keys():
     if 'v' in rkey:
          v = np.array(self.results[rkey])
```

**Comment by rgerkin** [comment]
Something like your fix attempted in 5c94837

**Comment by russelljjarvis** [comment]
I believe we resolved this one ages ago.
